Model/PaymentMethod/CreditCardPaymentMethod.php

```diff
@@ -553,6 +553,11 @@ public function addMandatoryTransactionData(&$oTransaction, $oOrder)
             ThreedsHelper::hasDownloadableItems($oBasket)
         );
 
+        if (!$this->getPayment()->getFieldData('wdoxidee_additional_info')) {
```

Comment:
why this if it needs to be added as mandatory? Also, what is line 561 again some logic with account holder?